assets/xml/objects/object_eg.xml

```diff
@@ -1,106 +1,154 @@
-        <Animation Name="object_eg_Anim_00D3D8" Offset="0xD3D8" />
-        <DList Name="object_eg_DL_00DCC0" Offset="0xDCC0" />
-        <Texture Name="object_eg_Tex_00DEA8" OutName="tex_00DEA8" Format="i4" Width="64" Height="64" Offset="0xDEA8" />
-        <Animation Name="object_eg_Anim_00EA10" Offset="0xEA10" />
-        <Animation Name="object_eg_Anim_00EE4C" Offset="0xEE4C" />
-        <Animation Name="object_eg_Anim_00F524" Offset="0xF524" />
-        <Animation Name="object_eg_Anim_00FB0C" Offset="0xFB0C" />
-        <Animation Name="object_eg_Anim_0100C0" Offset="0x100C0" />
-        <DList Name="object_eg_DL_010240" Offset="0x10240" />
-        <Texture Name="object_eg_Tex_010310" OutName="tex_010310" Format="rgba16" Width="36" Height="36" Offset="0x10310" />
-        <!-- <Blob Name="object_eg_Blob_010D30" Size="0x1000" Offset="0x10D30" /> -->
-        <Texture Name="object_eg_Tex_011D30" OutName="tex_011D30" Format="rgba16" Width="8" Height="16" Offset="0x11D30" />
-        <Texture Name="object_eg_Tex_011E30" OutName="tex_011E30" Format="rgba16" Width="16" Height="16" Offset="0x11E30" />
-        <Texture Name="object_eg_Tex_012030" OutName="tex_012030" Format="rgba16" Width="4" Height="4" Offset="0x12030" />
-        <Texture Name="object_eg_Tex_012050" OutName="tex_012050" Format="rgba16" Width="8" Height="16" Offset="0x12050" />
-        <Texture Name="object_eg_Tex_012150" OutName="tex_012150" Format="rgba16" Width="8" Height="8" Offset="0x12150" />
-        <Texture Name="object_eg_Tex_0121D0" OutName="tex_0121D0" Format="i4" Width="16" Height="32" Offset="0x121D0" />
-        <Texture Name="object_eg_Tex_0122D0" OutName="tex_0122D0" Format="rgba16" Width="16" Height="32" Offset="0x122D0" />
-        <!-- <Blob Name="object_eg_Blob_0126D0" Size="0xA20" Offset="0x126D0" /> -->
-        <Texture Name="object_eg_Tex_0130F0" OutName="tex_0130F0" Format="i4" Width="16" Height="32" Offset="0x130F0" />
+        <!-- Display lists and textures for Eyegore's effects -->
+        <DList Name="gEyegoreEffectImpactDL" Offset="0x40" />
+        <Texture Name="gEyegoreEffectImpactTex" OutName="eyegore_effect_impact" Format="ia8" Width="64" Height="64" Offset="0xC0" />
+        <TextureAnimation Name="gEyegoreEmpty1TexAnim" Offset="0x10C0" />
+        <DList Name="gEyegoreEffectLargeBodyPieceDL" Offset="0x1170" />
+        <Texture Name="gEyegoreEffectLargeBodyPieceTex" OutName="eyegore_effect_large_piece" Format="rgba16" Width="8" Height="16" Offset="0x1210" />
+        <DList Name="gEyegoreEffectSmallBodyPieceDL" Offset="0x13B0" />
+        <Texture Name="gEyegoreEffectSmallBodyPieceTex" OutName="eyegore_effect_small_piece" Format="rgba16" Width="8" Height="16" Offset="0x1450" />
+        <DList Name="gEyegoreEmpty1DL" Offset="0x1690" />
+
+        <!-- Assets for Eyegore's destructible pillars (unused) -->
+        <DList Name="gEyegoreBlockDL" Offset="0x1698" />
+        <Collision Name="gEyegoreBlockColHeader" Offset="0x1820" />
```

Comment:
I think there are a few stragglers in the codebase, but we prefer to suffix with `Col` instead of `ColHeader` for assets.
```suggestion
        <Collision Name="gEyegoreBlockCol" Offset="0x1820" />
```